php-frontend/src/test/java/org/sonar/php/cfg/ControlFlowGraphTest.java

```diff
@@ -76,6 +76,20 @@ public void break_without_argument() {
       "    break;" +
       "  }" +
       "}");
+
+    verifyBlockCfg("" +
+      "before(succ = [cond], elem = 1);" +
```

Comment:
as I said, I think `$i=1` should be part of this block (thus `elem = 2`)

Reply:
I'm moved the testing of `elem` in `simple_for` to avoid cluttering the tests for break, continue etc